x-pack/solutions/observability/plugins/slo/server/routes/slo/bulk_reset.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { BulkOperationStatusResponse } from '@kbn/slo-schema';
+import { bulkOperationParamsSchema, bulkOperationStatusParamsSchema } from '@kbn/slo-schema';
+import { v4 } from 'uuid';
+import { TYPE } from '../../services/tasks/bulk_reset/bulk_reset_task';
```

Comment:
nit: when I saw `TYPE` I was expecting it to be an enum, to select from it e.g., `TYPE.RESET`. It would be clearer to use the task type as a variable name? 🤔